pkgs/development/python-modules/skein/default.nix

```diff
@@ -10,6 +11,7 @@
 , hadoop
 , pytestCheckHook
 , python
+, versioneer
```

Comment:
```suggestion
, setuptools
, versioneer
```

Reply:
`setuptools` will need to be added to `propagatedBuildInputs` for python 3.12
```nix
propagatedBuildInputs = [ cryptography grpcio pyyaml ]
  ++ lib.optionals (!pythonOlder "3.12") [ setuptools ];
```

Reply:
Thanks for the review. Done.